opentelemetry/proto/common/v1/common.proto

```diff
@@ -79,3 +79,37 @@ message InstrumentationScope {
   repeated KeyValue attributes = 3;
   uint32 dropped_attributes_count = 4;
 }
+
+// A reference to an Entity.
+// Entity represents an object of interest associated with produced telemetry: e.g spans, metrics or logs.
+//
+// Status: [Development]
+message EntityRef {
```

Comment:
Both, class and field are marked as `Development` and put into the stable proto files. 

What is the strategy to release/version it?

Reply:
Great question - We documented the answer in https://github.com/open-telemetry/opentelemetry-proto/pull/597

If you need further details, happy to provide them.  

I will clarify, this proposal had a high-bar before opening the PR. We have three prototypes in different language SDKs (@dyladan just added javascript) and a Collector prototype.  So we're confident this is the right model going forward, but want to land the rest of the specification and SDK/Collector work before marking this as stable.

Reply:
@jsuereth, thank you. I was not aware about this PR.

Based on what I see, after release, the field(repeated opentelemetry.proto.common.v1.EntityRef entity_refs = 3;)  in resources will stay with definition forever (in the worst case as Obsolete).

I would like to just double check if it is the goal and confirm that it is mature enough decision.

Reply:
We've been prototyping this change for the past year, so yes - we consider this a mature enough decision to propose now.

We've been working with a fork of open-telemetry proto for quite a while as we explored this and have decided this is our path forward.  Otherwise I would not have raised this PR